src/main/java/dev/codedsakura/blossom/warps/WarpController.java

```diff
@@ -124,6 +126,18 @@ List<Warp> getWarps() {
         return data;
     }
 
+    List<Warp> getWarpableWarps(PlayerEntity player) {
+//        we know that the use must be permitted to run /warps to run /warps list
+        boolean isFPAPILoaded = Permissions.check(player, "blossom.warps.warps", false);
+        if (!isFPAPILoaded) {
+            return data;
+        } else {
+            return data.stream()
+                    .filter(warp -> Permissions.check(player, String.format("blossom.warps.warp.to.%s",warp.name), false))
```

Comment:
doesn't this contradict what you wrote in the readme? as this would default to false if FPAPI is loaded, right?